examples/rust/extend_viewer_ui/README.md

```diff
@@ -9,6 +9,10 @@ The example starts an SDK server which the Python or Rust logging SDK can connec
 
 ![image](https://github.com/rerun-io/rerun/assets/1148717/cbbad63e-9b18-4e54-bafe-b6ffd723f63e)
 
+Note that in order to spawn a web viewer with these customizations applied,
+you have to build the web viewer of the version yourself.
+This is currently not supported outside of the Rerun repository.
```

Comment:
We should make an issue for this. "Create tools for users to build their own web viewers" or similar

Reply:
* https://github.com/rerun-io/rerun/issues/2337